Quantum-Control-Applications-QuAM/Superconducting/pyproject.toml

```diff
@@ -1,9 +1,8 @@
 [tool.poetry]
```

Comment:
I recall some issues with poetry combined with an editable installation. In this case, static type checkers can't always resolve imports. We should verify that it doesn't cause issues in this case

Reply:
Let's sort the `pyproject.toml` discussion outside of this PR so that we can move on